python/cuml/CMakeLists.txt

```diff
@@ -98,6 +106,17 @@ if(NOT ${CUML_CPU})
 
     add_subdirectory(${CUML_CPP_SRC} cuml-cpp EXCLUDE_FROM_ALL)
 
+    if(NOT CUDA_STATIC_MATH_LIBRARIES AND USE_CUDA_MATH_WHEELS)
```

Comment:
Should we add an error if an incompatible combination of these two is specified? In theory you can use the static math libraries from the system and silently ignore `USE_CUDA_MATH_WHEELS` if both are true, but that seems confusing.

Reply:
I added an error if `USE_CUDA_MATH_WHEELS` is specified with CUDA < 12.0.